api_design/DDS-974-enforce_upload_chunk_size.md

```diff
@@ -54,7 +54,7 @@ The following properties will be added to the storage providers resource:
 `POST /projects/{id}/uploads`
 
 ###### Response Headers (Extensions)
-The following custom response headers will be added to inform clients of the minimum chunk size that may be utlized to ensure chunks can be coalesced, as well as the maximum chunk size the storage provider can accommodate.
+The following custom response headers will be added to inform clients of the minimum chunk size that may be utlized to ensure chunks can be coalesced when creating the maximum number of chunks, as well as the maximum chunk size the storage provider can accommodate.
```

Comment:
maybe say "inform clients of a suggested minimum chunk size that can be utilized ..."

Reply:
I don't understand what you want me to change here. Is it s/may/can/ ?

Reply:
I was suggesting the use of the word `suggested` :)
I just want to make sure people know that it is optional but recommended.

Reply:
Gotcha, that's a fair addition. I think I will also `s/can/will/` and `s/may/can/` while I'm at it.  :)